context/app/static/js/components/biomarkers/BiomarkersSearchBar.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import SearchBar from 'js/shared-styles/inputs/SearchBar';
+import { useBiomarkersSearchActions, useBiomarkersSearchState } from './BiomarkersSearchContext';
+
+const searchbarPlaceholder =
+  'Search for biomarkers by gene symbol, gene name or protein name. Example: CD4, Cytokeratin';
```

Comment:
If search is only gene symbol, change text to 
`Search for biomarkers by gene symbol. Note that searches are case-sensitive. Example: CD4, MMRN1`

If search is both gene name and symbol, change text to
`Search for biomarkers by gene name or symbol. Note that searches are case-sensitive. Example: CD4, MMRN1`

Reply:
I'm pending a response from Alan on this, but went with the former for now since that's the behavior we've personally observed